client/components/TextArea.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes, Component } from 'react';
+
+class TextArea extends React.Component {
```

Comment:
Because TextArea is a stateless component (it does not have its own this.state) we can refactor it look like the Button component